src/YEthStakerStrategy.sol

```diff
@@ -91,10 +92,17 @@ contract YEthStakerStrategy is
     function _deployFunds(uint256 _amount) internal override {
         if (_amount < WAD) {
             // no need to swap dust
+            // review: 1 eth is dust? that's a lot.
```

Comment:
Good call, we'll change this to variable `dust`.

Reply:
This is just to save gas, no need to deposit into yETH if there's less than 1 ETH deposited. Maybe the term "dust" is misleading here